hosts/azure/jenkins-controller/configuration.nix

```diff
@@ -149,6 +149,32 @@ in {
             };
           };
         }
+        {
+          # in order to get this pipeline cron activated first time, one needs to initiate one build manually from UI
```

Comment:
Fix formatting, also see the CI failure: https://github.com/tiiuae/ghaf-infra/actions/runs/9127052904/job/25096544059?pr=151#step:4:524

Reply:
A bit unclear, cleaned comment of. This has been running two days in my Ghaf-infra...